backend/services/implementations/classService.ts

```diff
@@ -94,6 +94,26 @@ class ClassService implements IClassService {
     return (await this.mapClassToClassDTOs(classes))[0];
   }
 
+  async getClassesByTeacherId(
+    teacherId: string,
+  ): Promise<Array<ClassResponseDTO>> {
+    let classes: Class[];
+    try {
+      classes = await MgClass.find({ teacher: { $eq: teacherId } });
+      if (!classes.length) {
+        return [];
+      }
```

Comment:
```suggestion
```
actually i think .find() directly returns [] so you don't need this

Reply:
true true